src/keyboard/emacs.js

```diff
@@ -105,6 +105,14 @@ exports.handler.attach = function(editor) {
     editor.on('paste', this.onPaste);
 };
 
+function pushUnique(ring, mark) {
+    var last = ring[ring.length - 1];
+    if (last && last.row == mark.row && last.column == mark.column) {
```

Comment:
nit: should we use `===` here? or is the datatype flexible here, allowing both strings and numbers?

Reply:
These are always numbers, it's just an old time habit of mine to use `==` if linter is allowing that.